client/jetpack-cloud/sections/agency-dashboard/sites-overview/style.scss

```diff
@@ -0,0 +1,136 @@
+@import '@wordpress/base-styles/breakpoints';
+@import '@wordpress/base-styles/mixins';
+
+.sites-overview {
+	height: 100vh;
+	padding: 16px;
+	@include break-large() {
+		padding: 0;
+	}
+	@include break-wide() {
+		padding: 16px;
+	}
+}
+.sites-overview__status-add-new {
+	display: inline-flex;
+	flex-direction: row;
+	justify-content: center;
+	align-items: center;
+	width: fit-content;
+	height: 22px;
+	background: var( --studio-white );
+	box-sizing: border-box;
+	border-radius: 12px; /* stylelint-disable-line scales/radii */
+	font-weight: 500;
+	font-size: 0.75rem;
+	vertical-align: middle;
+	color: var( --studio-gray-80 );
+	border: 1px solid var( --studio-gray-5 );
+	padding: 2px 11px;
+	span {
+		margin: 0 0.2em;
+	}
+	svg {
+		margin-left: -0.4em;
+	}
+	&:hover {
+		background: var( --studio-gray-80 );
+		color: var( --studio-white );
+	}
+}
+.sites-overview__grey-icon {
+	vertical-align: middle;
+	color: var( --studio-gray-40 );
+}
+.sites-overview__icon-active {
+	vertical-align: middle;
+	color: var( --studio-gray-5 );
+}
+.sites-overview__disabled {
+	color: var( --studio-gray-5 );
+	cursor: not-allowed;
+	opacity: 0.5;
+}
+.sites-overview__row-text {
+	display: inline-block;
+	font-weight: 500;
+	white-space: nowrap;
+	overflow: hidden;
+	text-overflow: clip;
+	vertical-align: middle;
+	color: var( --studio-gray-100 );
+	@include break-large() {
+		width: 100%;
+		margin-left: 0;
+		font-size: 0.875rem;
+	}
+}
+.sites-overview__vertical-align-middle {
+	vertical-align: middle;
+}
+.sites-overview__error-container {
+	background: #414141;
+	margin: 0 -6px;
+	display: flex;
+	align-items: center;
+	height: 40px;
+	@include break-large() {
+		margin: 8px;
+	}
+}
+.sites-overview__error-icon {
+	background: #d94f4f;
+	padding: 11px;
+	color: var( --studio-white );
+	width: 5%;
+	display: flex;
+	align-items: center;
+	justify-content: center;
+	@include break-large() {
+		width: auto;
+	}
+}
+.sites-overview__error-message {
+	font-size: 0.75rem;
+	color: var( --studio-white );
+	padding:0.5em;
+	margin: auto 0;
+}
+.sites-overview__error-message-large-screen {
+	display: inline-block;
+}
+.sites-overview__error-message-link {
+	font-size: 0.75rem;
+	color: #bbbbbb!important;
+	padding: 6px;
+	position: absolute;
+	right: 16px;
+}
+.sites-overview__badge {
+	font-size: 0.75rem!important;
+	white-space: nowrap;
+	overflow: hidden;
+	text-overflow: ellipsis;
+	vertical-align: middle;
+	@include break-large() {
+		max-width: 70px;
+	}
+	@include break-wide() {
+		max-width: fit-content;
+	}
+}
+.sites-overview__tooltip {
+	.popover__arrow {
+		&::before {
+			border-bottom-color: var( --studio-gray-60 )!important;
+			top: 1px!important;
```

Comment:
Can we include space before all `!important` usages? Although it's not required, for me, that would improve readability a bit. Also, depending on the search settings (e.g., match full word), the IDEs might skip that from the results.